lib/css/modules/_accountSwitcher.scss

```diff
@@ -1,8 +1,10 @@
 #RESAccountSwitcherDropdown {
-	min-width: 110px;
-	width: auto;
 	display: none;
+	height: 400px;
```

Comment:
If this is replaced by `height: calc(100vh - 80px)` , then you avoid unnecessarily limiting the height of the account switcher.